test/mysql.discover.test.js

```diff
@@ -206,6 +206,42 @@ describe('Discover model foreign keys', function() {
   });
 });
 
+describe('Discover model generated columns', function() {
+  it('should return an array of columns for STRONGLOOP.PRODUCT and none of them is generated', function(done) {
```

Comment:
Actually what your saying is incorrect. An array of columns is returned regardless. I'm testing here that none of the columns in the array have generated=true. The description of the test should express that. I could omit STRONGLOOP.PRODUCT in the description if you find that important, but as you can see all the other tests are expressed just the same.